src/content/en/shows/designer-vs-developer/podcast/DVDS2E02-designing-a-great-performance.md

```diff
@@ -0,0 +1,54 @@
+project_path: /web/_project.yaml
+book_path: /web/shows/_book.yaml
+description: Addy talks about designing performance.
+
+{# wf_updated_on: 2017-08-07 #}
+{# wf_published_on: 2017-08-07 #}
+{# wf_podcast_audio: https://storage.googleapis.com/designer-vs-developer-podcast/DVDS2E02-designing-a-great-performance.mp3 #}
+{# wf_podcast_duration: 00:13:34 #}
+{# wf_podcast_fileSize: 19566017 #}
+{# wf_podcast_subtitle: Addy talks about designing performance. #}
+{# wf_featured_image: /web/shows/designer-vs-developer/podcast/images/dvd-s02-e02-cover-large.jpg #}
+
+# Designing a great performance {: .page-title }
```

Comment:
'a great performance'. If I'm inferring what you're trying to say, it should be 'for great performance'.